libvast/src/directory.cpp

```diff
@@ -19,6 +19,8 @@
 #include "vast/detail/posix.hpp"
 #include "vast/detail/string.hpp"
 
+#include "vast/logger.hpp"
```

Comment:
This doesn't look properly formatted, this should be joined with the group above.